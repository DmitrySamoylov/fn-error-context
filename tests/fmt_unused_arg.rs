use fn_error_context::error_context;

#[error_context("context", 1)]
fn do_stuff() -> anyhow::Result<()> {
    anyhow::bail!("error")
}

fn main() {}
