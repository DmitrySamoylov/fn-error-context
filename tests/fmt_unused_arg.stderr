error: argument never used
 --> $DIR/fmt_unused_arg.rs:3:28
  |
3 | #[error_context("context", 1)]
  |                 ---------  ^ argument never used
  |                 |
  |                 formatting specifier missing
